desktop/hardened-desktop/admin-controls/configure-ac.md

```diff
@@ -0,0 +1,130 @@
+
+- Mac: `/Library/Application Support/com.docker.docker/admin-settings.json`
+- Windows: `/ProgramData/DockerDesktop/admin-settings.json`
+- Linux: `/usr/share/docker-desktop/admin-settings.json`
+
+By placing this file in the above protected directories, end users are unable to modify it.
+
+>Note
+>
+> It is assumed that you have the ability to push the `admin-settings.json` settings file to the locations specified above through a device management software such as [Jamf](https://www.jamf.com/lp/en-gb/apple-mobile-device-management-mdm-jamf-shared/?attr=google_ads-brand-search-shared&gclid=CjwKCAjw1ICZBhAzEiwAFfvFhEXjayUAi8FHHv1JJitFPb47C_q_RCySTmF86twF1qJc_6GST-YDmhoCuJsQAvD_BwE).
+
+## Step two: Configure the admin controls you want to lock in
+
+>Note
+>
+>Some of the configuration parameters only apply to Windows. This is highlighted in the table below.
+
+The `admin-settings.json` file requires a nested list of configuration parameters, each of which must contain the  `locked` parameter. You can add or remove configuration parameters as per your requirements.
+
+If `locked: true`, users are not able to edit this setting from Docker Desktop or the CLI.
+
+If `locked: false`, it is similar to setting a factory default in that:
+- For new installs, `locked: false` pre-populates the relevant settings in the Docker Desktop UI, but users are able to modify it.
+
+- If Docker Desktop is already installed and being used, `locked: false` is ignored. This is because existing users of Docker Desktop may have already updated a setting, which in turn will have been written to the relevant config file, for example the `settings.json` or `daemon.json`. In these instances, the user's preferences are respected and we don't alter these values. These can be controlled by the admin by setting `locked: true`.
+
+The following `admin-settings.json` code and table provides an example of the required syntax and descriptions for parameters and values:
+
+```json
+{
+  "configurationFileVersion": 2,
+  "exposeDockerAPIOnTCP2375": {
+    "locked": true,
+    "value": false
+  },
+  "proxy": {
+    "locked": true,
+    "mode": "manual",
+    "server": "myproxy.com",
+    "port":3129,
+    "exclude": ["foo.com", "bar.com"]
+  },
+  "enhancedContainerIsolation": {
+    "locked": true,
+    "value": true
+  },
+  "useWindowsContainers": {
+      "locked": false,
+      "value": false
+  },
+  "linuxVM": {
+    "wslEngineEnabled": {
+      "locked": false,
+      "value": false
+    },
+    "dockerDaemonOptions": {
+      "locked": false,
+      "value":"{\"debug\": false}"
+    },
+    "vpnkitCIDR": {
+      "locked": false,
+      "value":"192.168.65.0/24"
+    }
+  },
+  "windowsContainers": {
+    "dockerDaemonOptions": {
+      "locked": false,
+      "value":"{\"debug\": false}"
+    }
+  },
+  "disableUpdate": {
+    "locked": false,
+    "value": false
+  },
+  "analyticsEnabled": {
+    "locked": false,
+    "value": true
+  },
```

Comment:
@aevesdocker can you remove the trailing comma here as this makes the JSON invalid ?